packages/synapse-react-client/src/lib/containers/markdown/widget/MarkdownButton.tsx

```diff
@@ -21,15 +22,18 @@ export default function MarkdownButton(
     buttonClasses += 'floatright '
   }
   const buttonIsCenterAligned = alignLowerCase === 'center'
-  const buttonVariant = highlight === 'true' ? 'secondary' : 'inherit'
+  const buttonColor = highlight ? 'secondary' : 'neutral'
```

Comment:
`'inherit'` isn't a real button color, this is a holdover from bootstrap that doesn't make sense, even if it still worked. Replace with a new color palette 'neutral'